src/core_modules/capture-core/components/WidgetErrorAndWarning/content/WidgetErrorAndWarningContent.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+
+import React from 'react';
+import { spacers, colors, spacersNum } from '@dhis2/ui';
+import { withStyles } from '@material-ui/core';
+import cx from 'classnames';
+import type { contentTypes, renderObjectType, renderStringType } from './WidgetErrorAndWarningContent.types';
+
+const styles = {
+    widgetWrapper: {
+        padding: `0px ${spacersNum.dp16}px`,
+    },
+    unorderedList: {
+        padding: `0px ${spacers.dp16}`,
+        marginTop: '0px',
+        lineHeight: '1.375',
+        fontSize: spacers.dp16,
+        fontWeight: 400,
+        color: colors.grey900,
+        display: 'flex',
+        flexDirection: 'column',
+        gap: spacers.dp12,
+    },
+    listItem: {
+        '&.error': {
+            listStyleType: '"!   "',
+            '&::marker': {
+                color: colors.red800,
+                fontWeight: 700,
+            },
+        },
+        '&.warning': {
+            '&::marker': {
+                color: colors.yellow900,
+                fontWeight: 700,
+            },
+        },
+    },
+};
+
+const RenderObjectItem = ({ rule, listItem }: renderObjectType) => (
```

Comment:
NIT: I think the name of JSX component should be only `ObjectItem` since `RenderObjectItem` is a verb and refer to function

Reply:
Yup, good point 👍